src/iptvsimple/Settings.h

```diff
@@ -138,6 +146,11 @@ namespace iptvsimple
     int m_epgTimeShiftMins = 0;
     bool m_tsOverride = true;
 
+    bool m_useEpgGenreTextWhenMapping = false;
+    PathType m_genresPathType = PathType::LOCAL_PATH;
+    std::string m_genresPath = "";
+    std::string m_genresUrl = "";
```

Comment:
No need to init the std::string members with empty string as this is what the default std::string ctor does anyway.